assembl/static2/js/app/components/form/datePickerFieldAdapter.jsx

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+/*
+  DatePicker adapter for react-final-form that supports the following form:
+  { time: moment().utc() }
+
+  Errors are managed by a Mutator on the Form itself. As a result, to show errors
+  on DatePickerFieldAdapter, must add the `setFieldTouched` from final-form-set-field-touched
+  along with passing the form <FormApi> prop
+*/
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import * as React from 'react';
+import { type FieldRenderProps } from 'react-final-form';
```

Comment:
@ayyazdaniaryan could you regroup every similar import altogether: component with component, type with type and so on. Thanks !